lib/plugins/aws/package/compile/events/http-api.js

```diff
@@ -630,9 +625,6 @@ Object.defineProperties(
         IntegrationUri: resolveTargetConfig(routeTargetData),
         PayloadFormatVersion: funcHttpApi.payload || providerHttpApi.payload || '2.0',
       };
-      if (routeTargetData.timeout) {
-        properties.TimeoutInMillis = Math.round(routeTargetData.timeout * 1000);
-      }
```

Comment:
Is `routeTargetData.timeout` also coming from the `serverless.yml` config?

Said another way: can users set the route timeout via serverless.yml? If we remove these lines, will this break?

Reply:
I checked the history. I think `routeTargetData.timeout` was also coming from the serverless.yml config but was deprecated